src/Essentials/src/FileSystem/FileSystem.shared.cs

```diff
@@ -87,6 +128,20 @@ internal static string Clean(string extension, bool trimLeadingPeriod = false)
 				return extension;
 			}
 		}
+
+#nullable enable
```

Comment:
The nullable scope should also cover Current, so ight as well use a single block.